blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_7_serviceability_limit_state.py

```diff
@@ -0,0 +1,49 @@
+"""Module including all formulas from chapter 7 of NEN-EN 1992-1-1+C2:2011."""
+# pylint: disable=arguments-differ
+from blueprints.codes.formula import Formula
+from blueprints.typehints import KN, KN_TO_N, MM2, MPA
+
+
+class Form7Dot3CoefficientKc(Formula):
+    """Class representing the formula 7.3 for the coefficient kc for flanges of tubular cross-sections and T-sections [-]."""
+
+    label = "7.3"
+    source_document = "NEN-EN 1992-1-1+C2:2011"
+
+    def __init__(
+        self,
+        f_cr: KN,
+        a_ct: MM2,
+        f_ct_eff: MPA,
+    ) -> None:
+        """Calculates kc for flanges of tubular cross-sections and T-sections [-].
+
+        NEN-EN 1992-1-1:2011 art.7.3
+
+        Parameters
+        ----------
+        f_cr : KN
+            Absolute value of the tensile force within the flange immediately before cracking due to the cracking moment calculated with fct,eff [kN].
+        a_ct : MM2
+            Area of the concrete within the tension zone. The tension zone is that part of the cross-section that, according to the calculation,
+            is under tension just before the first crack occurs [mm²].
+        f_ct_eff : MPA
+            Average value of the tensile strength of the concrete at the time when the first cracks can be expected [MPa].
+        """
+        super().__init__()
+        self.f_cr = f_cr
+        self.a_ct = a_ct
+        self.f_ct_eff = f_ct_eff
+
+    @staticmethod
+    def _evaluate(
+        f_cr: KN,
+        a_ct: MM2,
+        f_ct_eff: MPA,
+    ) -> float:
+        """For more detailed documentation see the class docstring."""
```

Comment:
what is the convention we use for the _evaluate method? It is actually part of the init as it is private and activate in the init. However two docstrings are a bit overkill. 
Do we add a default docstring to the def _evaluate method which is always the same? 
`
"""Evaluates the formula, for more infor see the __init__ method"""
`

Reply:
I wouldn't even mind if the static evaluate would be public, that gives you the chance to run the function lightweight if you are only interested in a 'float' answer.

About sharing docstrings, maybe we can get some inspiration from this thread: https://softwareengineering.stackexchange.com/questions/386755/sharing-docstrings-between-similar-functions

Reply:
Yes, the docstring of the parameters for the formula are already defined in the __init__